miracl/flow/miracl_workflow_ace_parser.py

```diff
@@ -688,10 +688,9 @@ def parsefn(self) -> argparse.ArgumentParser:
             "-ua",
             "--u_atlas_dir",
             default="miracl_home",
-            help="path of atlas directory (default: '/code/atlases/ara/')"
+            help="path of atlas directory (default: '/code/atlases/ara/')",
```

Comment:
I looked into it. So `miracl_home` is assigned to nothing but `MIRACL_HOME` is `/code/miracl`. Our script will perform a check to see if this arg equals `miracl_home` and then it will set the atlas dir to `/code/atlases/ara/`. I will fix this to be more programatic.

Reply:
Great, thanks a lot! That would make things a lot cleaner. Would it make sense for us to have an atlas env var like `ATLAS_HOME` or sth? It feels like `MIRACL_HOME` should refer to `/code/miracl` and not the atlases. Or would that take too much effort to change in this PR? I'm currently implementing the Fischer rat atlas so an env var to the atlas base dir could be useful.

Reply:
I guess we already have it in the `Dockerfile` under `aradir`:
https://github.com/AICONSlab/MIRACL/blob/79249afd07fb7872712466135c7d892ded810c18/Dockerfile#L59

Reply:
Love the solution you came up with for this in 0018b20! I'm going to add another env var to the Dockerfile for the Fischer atlas that we can reference!

Reply:
I also just noticed that we have `$ATLASES_HOME` in the Dockerfile. I'm using that for the Fischer implementation but also added `$fischerdir` as well.

Reply:
I think eventually we should have a utils file with all the constants like `ARA_ENV`, `FISCHER_ENV` so we can import from there and not have to define it in each file.